gitgeo/custom_csv.py

```diff
@@ -17,7 +17,9 @@ def create_csv(results_type, timestamp):
     Returns:
         None
     """
-    filename = os.path.join("results", results_type + "_" + timestamp + ".csv")
+    filename = Path.cwd() / "results"
+    filename.mkdir(exist_ok=True)
+    filename /= results_type + "_" + timestamp + ".csv"
```

Comment:
@jbenjoseph, one question here: What is this ```\=``` in the context of a string operation? Google failed me here.

Reply:
This is overloaded by the ```pathlib``` for creating paths in a cross-platform manner. Using ```/=``` is equivalent to  ```filename = filename / results_type + "_" + timestamp + ".csv"```